week08_pomdp/practice_pytorch.ipynb

```diff
@@ -444,12 +466,50 @@
    "source": [
     "def to_one_hot(y, n_dims=None):\n",
     "    \"\"\" Take an integer tensor and convert it to 1-hot matrix. \"\"\"\n",
-    "    y_tensor = y.to(dtype=torch.int64).view(-1, 1)\n",
+    "    y_tensor = y.to(dtype=torch.int64).contiguous().view(-1, 1)\n",
     "    n_dims = n_dims if n_dims is not None else int(torch.max(y_tensor)) + 1\n",
     "    y_one_hot = torch.zeros(y_tensor.size()[0], n_dims).scatter_(1, y_tensor, 1)\n",
     "    return y_one_hot"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Build Reproducibility function"
```

Comment:
Um, what does this mean?

Reply:
**Reproducibility Function** is one of the significant functions in AI since we depend on random values generated every time in our network and some of our sampled actions. So, Having a reproducibility function helps to ensure that the generated objects will be the same every time. (a.k.a this what we call it **seeds**).